dotCMS/src/main/java/com/dotmarketing/common/db/DotDatabaseMetaData.java

```diff
@@ -546,19 +546,26 @@ private List<String> getColumnConstraintsMSSQL (final Connection connection, fin
     }
 
     public void dropIndex(final String tableName, final String indexName) throws SQLException {
+        try (Connection conn = DbConnectionFactory.getConnection()) {
```

Comment:
Is this try-with-resources needed ?

Reply:
yeah. After what I learned today I guess it is not.